src/pugicast.h

```diff
@@ -20,18 +20,13 @@
 #ifndef FS_PUGICAST_H_07810DF7954D411EB14A16C3ED2A7548
 #define FS_PUGICAST_H_07810DF7954D411EB14A16C3ED2A7548
 
-#include <boost/lexical_cast.hpp>
-
 namespace pugi {
 	template<typename T>
 	T cast(const pugi::char_t* str)
 	{
 		T value;
-		try {
-			value = boost::lexical_cast<T>(str);
-		} catch (boost::bad_lexical_cast&) {
-			value = T();
-		}
+		std::stringstream ss(str);
+		ss >> value;
```

Comment:
@marksamman is this acceptable as a lexical cast replacement? If extraction fails, `value` will be left unchanged (in this case, will be a default-constructed value) so it should perform the same conversion. [(source)](http://en.cppreference.com/w/cpp/io/basic_istream/operator_gtgt)

The same happens on DBResult, and `operator>>` has a lot of overloads.